File: backend/src/main/java/com/bryan/system/domain/request/user/UserExportRequest.java
package com.bryan.system.domain.request.user;

import lombok.Data;

import java.util.List;

/**
 * 用户导出请求对象
 *
 * @author Bryan Long
 */
@Data
public class UserExportRequest {
    /**
     * 要导出的字段列表
     */
    private List<String> fields;

    /**
     * 导出文件名（可选，默认为"用户数据"）
     */
    private String fileName;

    /**
     * 状态过滤（可选）
     * 0: 正常, 1: 封禁
     */
    private Integer status;
}

File: frontend/src/types/request/user/UserExportRequest.ts
/**
 * @module src/types/request/user/UserExportRequest.ts
 * @description 定义用户数据导出请求的数据结构。
 */
export interface UserExportRequest {
    fields?: string[]; // 要导出的字段列表
    fileName?: string; // 导出文件名
    status?: number; // 状态过滤
}